_posts/2018-08-13-Introducing....md

```diff
@@ -0,0 +1,96 @@
+
+#### “Tell me about your research at the Turing”
+
+Ang is interested in mental disorders, especially schizophrenia, and developing clinically useful biomarkers to help psychiatrists by using genomic or neuroimaging data. Ang would also like to use genomic data to understand the pathology of schizophrenia. The pathology itself is complex, but it is a very important area to understand, and many answers emerge from examining the genome. The other part of Ang’s research is about understanding the concept of human consciousness. Ang is also interested in neurodevelopment, using neuroimaging data to help understand the emergence and pathologies of some mental disorders– this could be used to predict why adolescents might have mental health challenges.
+
+#### “What’s your ideal outcome of your time at the Turing?”
+
+Ang would like to learn about reproducible research from Kirstie. His area normally deals with many different types of data, for instance genome data, gene expression and MRI (magnetic resonance imaging) data. As the real real-world questions, especially in brain science, are always complex, expertise from a lot of different domains is required to answer them: psychology, psychiatry, genomic science, mathematics and even physics. Ang believes that a diverse community of people from different specialisms is a necessary to answer some important scientific question. And it could be useful to cooperate by linking their ideas closely along with some uniform code.
+
+#### “How are you finding the Turing?”
+
+“I love the working environment of the Turing, it’s different from my Chinese lab and I have more free space – I can work in a different place every day. Also, the Turing has the great advantage that we can meet people with diverse backgrounds and fields.”
+
+## :star: Yini He :star: 
+
+#### “Tell me about your background” 
+
+Yini majored in psychology and is currently doing a PhD in neuroimaging. She changed her subject because during the last year of her undergraduate degree she started to be an intern teacher in a special school, and a lot of students had mental health challenges such as depression, bipolar disorder, and schizophrenia. Yini found her time teaching them an amazing experience. She enjoyed their creativity and talking to them, and she wanted to do things to help them and to change their lives for the better. She couldn’t find a perfect way to do that within education and psychology, so she decided to change her field. She started working in a hospital - Xiangya – which is very famous hospital in China – and met people there. She realised she would like to know about adolescents, and to research if it would be possible to screen vulnerability to mental health issues, and maybe use the screening to find ways to make people’s lives better in the future. 
+
```

Comment:
Yini majored in psychology and is currently doing a PhD in neuroimaging. She changed her subject because during the last year of her undergraduate degree she started to be an intern teacher in a special school, and a lot of students had mental health challenges such as depression, bipolar disorder, and schizophrenia. Yini found her time teaching them an amazing experience. She enjoyed their creativity and talking to them, and she wanted to do things to help them and to change their lives for the better. **She thought she had to know more about neural mechanisms of mental disorders in order to gain a deeper understanding of them**,so she decided to change her field. She started working in a hospital - Xiangya – which is very famous hospital in China – and met people there. She realised she would like to know about adolescents, and to research if it would be possible to screen vulnerability to mental health issues, and maybe use the screening to find ways to make people’s lives better in the future.